### DISCUSSION TITLE
edit environment

### DISCUSSION THREAD

**Discussion by hunter55555** [harfang3d/harfang3d#46] [Q&A]
hello， thanks for your work, it's very useful for us. I want to edit the environment and add new elements such as a boat or car in the custom air combat environment, but I don't know how to do it, the website can't download the studio, additionally, this is how to write API to call by an algorithm. I look forward to your reply. Thanks again.

**Comment by astrofra** [comment]
Hi, sorry for the inconvenience, we are fixing the issue on the website, thanks for mentioning it.
In the meantime, you can download HARFANG Studio from this URL : https://github.com/harfang3d/harfang3d/releases/download/v3.2.5/studio-0.8.3-win64.exe

**Reply by hunter55555** [reply]
Your reply was so timely, I am full of gratitude to you for helping me.
